examples/src/bin/direct_vectoring.rs

```diff
@@ -42,17 +41,24 @@ fn main() -> ! {
         );
     }
 
-    esp_println::println!("MPC:{}", unsafe { fetch_performance_timer() });
+    #[cfg(not(any(feature = "esp32c6", feature = "esp32h2")))]
+    let system = unsafe { &*esp_hal::peripherals::SYSTEM::PTR };
```

Comment:
I really don't want to encourage this :/

Reply:
Let me see if I can find a better way

Reply:
I avoided the unsafe, but couldn't avoid the cfg because the registers are associated with different peripheral blocks.